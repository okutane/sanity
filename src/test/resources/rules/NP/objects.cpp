struct Foo {
    ~Foo();
    void bar();
    virtual void baz();
} *GPFOO;

static void testNonVirtual() {
    GPFOO = 0;
    GPFOO->bar();
}

static void testVirtual() {
    GPFOO = 0;
    GPFOO->baz();
}