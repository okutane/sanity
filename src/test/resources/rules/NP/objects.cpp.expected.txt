Violation: tmp0 is null
 - no source -
 - no source -
 - no source -

